lf_steer_node:
  ros__parameters:
    channel: 0
    servo_type: standard
    min: 0
    max: 180
    neutral: 90
    subscription_nodes: [
      ackerman_drive_node_lf_steer_node_pub
    ]
lf_drive_node:
  ros__parameters:
    channel: 1
    servo_type: continuous
    min: 0
    max: 180
    neutral: 90
    subscription_nodes: [
      ackerman_drive_node_lf_drive_node_pub
    ]
  